roles/podman/handlers/main.yml

```diff
@@ -0,0 +1,4 @@
+---
+- name: reload systemd
+  systemd:
+    daemon_reload: true
```

Comment:
For sake of correctness, this should probably be in a role called `systemd` with just this handler that's depended upon, otherwise we may end up duplicating this everywhere.